libs/core/src/lib/utils/drag-and-drop/dnd-item/dnd-item.directive.ts

```diff
@@ -0,0 +1,216 @@
+import { AfterContentInit, Directive, ElementRef, EventEmitter, Input, Output } from '@angular/core';
+import { CdkDragMove, DragDrop, DragRef } from '@angular/cdk/drag-drop';
+import { ElementChord, LinkPosition } from '../dnd-list/dnd-list.directive';
+
+@Directive({
+    // tslint:disable-next-line:directive-selector
+    selector: '[fd-dnd-item]',
+    host: {
+        class: 'fd-dnd-item'
+    },
+    providers: [
+        DragDrop
+    ]
+})
+export class DndItemDirective implements AfterContentInit {
+    /** Event thrown when the element is moved by 1px */
+    @Output()
+    readonly moved = new EventEmitter<CdkDragMove>();
+
+    /** Event thrown when the element is released */
+    @Output()
+    readonly released = new EventEmitter<void>();
+
+    /** Event thrown when the element is started to be dragged */
+    @Output()
+    readonly started = new EventEmitter<void>();
+
+    /** Whether this element should stick in one place, without changing position */
+    @Input()
```

Comment:
Do we need both `stickInPlace` and `dndDisabled`? They seem to define the same behavior. Maybe we could replace it with one `draggable` input.

Reply:
Yes, It's needed cause for example first item with `stickInPlace` can't have dragged any element before it's position. I can see case, where someone blocks element from being dragged, but makes placing elements before, or after this element possible.

Reply:
- Can we make it clear in the documentation? That part saying that `stickInPlace` prevents the item to be dragged and moved by the other elements.

- I think we should also make it possible to turn off and on the dragging. Right now its only set-up on `ngAfterContentInit` so there is no way to change that behavior after its set at the beginning.

- What do you think about renaming it into `draggable`? `dndDisabled` is something that I'd rather expect to see on the container, disabling and enabling draggable behavior in for all of the elements in the container (maybe we should also add this feature?).

Reply:
The disabled state for whole list is added. Also naming has been changed. I updated comments